WordPress/src/main/java/org/wordpress/android/WordPressDB.java

```diff
@@ -371,6 +388,11 @@ private void deleteUploadedLocalDrafts() {
         }
     }
 
+    private void deleteThemesTable() {
```

Comment:
Odd method name considering it re-creates the table. Maybe `migrateThemesTable()`?